app.js

```diff
@@ -0,0 +1,98 @@
+const createError = require('http-errors');
+const express = require('express');
+const path = require('path');
+const cookieParser = require('cookie-parser');
+const logger = require('morgan');
+const mangoose = require('mongoose');
+
+const swaggerUI = require('swagger-ui-express');
+const swaggerYaml = require('yamljs');
+
+const indexRouter = require('./routes/index');
+const usersRouter = require('./routes/users');
+
+const dirname = path.resolve();
+
+mangoose
+  .connect("mongodb://localhost:27017/tutorials", {
+    useNewUrlParser: true,
+  })
+  .then(() => console.log("Connection successfull!"))
+  .catch((err) => console.log);
+
+// Schema
+const tutorialSchema = new mangoose.Schema({
```

Comment:
Define this model ina seperate file.
Example:
```
app\
  routes\
  export all the models from here
```

CC: @JainishaParmar @jaimin45 @ashishkava